packs/next.js

```diff
@@ -0,0 +1,22 @@
+
+const icon = `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 207 124'%3E%3Cg fill='%23000' fill-rule='nonzero'%3E%3Cpath d='M48.942 32.632h38.96v3.082H52.512v23.193h33.278v3.082H52.513v25.464h35.794v3.081H48.942V32.632Zm42.45 0h4.139l18.343 25.464 18.749-25.464L158.124.287l-41.896 60.485 21.59 29.762h-4.302l-19.642-27.086L94.15 90.534h-4.22l21.751-29.762-20.29-28.14Zm47.967 3.082v-3.082h44.397v3.082h-20.453v54.82h-3.571v-54.82h-20.373ZM.203 32.632h4.464l61.557 91.671-25.439-33.769L3.936 37.011l-.162 53.523H.203zM183.397 86.523c.738 0 1.276-.563 1.276-1.29 0-.727-.538-1.29-1.276-1.29-.73 0-1.277.563-1.277 1.29 0 .727.547 1.29 1.277 1.29Zm3.509-3.393c0 2.146 1.555 3.549 3.822 3.549 2.414 0 3.874-1.446 3.874-3.956v-8.837h-1.946v8.828c0 1.394-.704 2.138-1.946 2.138-1.112 0-1.867-.692-1.893-1.722h-1.911Zm10.24-.113c.14 2.233 2.007 3.662 4.787 3.662 2.97 0 4.83-1.498 4.83-3.887 0-1.878-1.06-2.917-3.632-3.514l-1.38-.338c-1.634-.38-2.294-.891-2.294-1.783 0-1.125 1.025-1.86 2.563-1.86 1.459 0 2.466.718 2.649 1.869h1.893c-.113-2.103-1.971-3.583-4.516-3.583-2.737 0-4.56 1.48-4.56 3.704 0 1.835 1.033 2.926 3.3 3.454l1.616.39c1.659.389 2.388.96 2.388 1.912 0 1.108-1.146 1.913-2.71 1.913-1.676 0-2.84-.753-3.005-1.939h-1.928Z'/%3E%3C/g%3E%3C/svg%3E`;
+
+const UIStrings = {
+  'unused-css-rules': 'PurgeCSS can be used in Next.js to remove unused rules from stylesheets. [Learn more](https://purgecss.com/guides/next.html).',
+  'modern-image-formats': 'Use Next.js Image Optimization API to serve modern formats like WebP and AVIF. [Learn more](https://nextjs.org/docs/api-reference/next/image#acceptable-formats).',
+  'offscreen-images': 'Use Next.js Image component, `next/image`, which defaults to `loading="lazy"`. [Learn more](https://nextjs.org/docs/api-reference/next/image#loading).',
+  'render-blocking-resources': 'Use Next.js Script component, `next/script`, to defer loading of non-critical third-party scripts. [Learn more](https://nextjs.org/docs/basic-features/script).',
```

Comment:
```suggestion
  'render-blocking-resources': 'Use the `next/script` component to defer loading of non-critical third-party scripts. [Learn more](https://nextjs.org/docs/basic-features/script).',
```

this is relevant to all entries:

1) adding definite article `the`
2) Drop "x component, \`y\`" and prefer "\`y\` component", which avoids potential translation confusion. I am not sure if component should be translated, so I left it out of the backticks. Do next.js tech writers have any preference?